src/Sentry/Protocol/SentryId.cs

```diff
@@ -29,20 +29,39 @@ namespace Sentry
         /// dashes which sentry doesn't expect when searching events.
         /// </remarks>
         /// <returns>String representation of the event id.</returns>
-        public override string ToString() => _eventId.ToString("n");
+        public override string ToString() => _guid.ToString("n");
+
+        // Note: spans are sentry IDs with only 16 characters, rest being truncated.
+        // This is obviously a bad idea as it invalidates GUID's uniqueness properties
+        // (https://devblogs.microsoft.com/oldnewthing/20080627-00/?p=21823)
+        // but all other SDKs do it this way, so we have no choice but to comply.
+        /// <summary>
+        /// Returns a truncated ID.
+        /// </summary>
+        public string ToShortString() => ToString().Substring(0, 16);
```

Comment:
Lets please discuss this before taking this approach.

Why do we need to do this, because other SDKs did? That's not a reason.

/cc @marandaneto

Reply:
Can we also not capture the transaction directly on `Transaction.Finish()`? 😄 
That's how other SDKs did it but it introduces a lot of complexity.